satpy/multiscene.py

```diff
@@ -490,8 +521,56 @@ def _simple_frame_compute(self, writers, frame_keys, frames_to_write):
                 w = writers[frame_key]
                 w.append_data(product_frame.compute())
 
+    def _get_writers_and_frames(
+            self, filename, datasets, fill_value, ignore_missing,
+            enh_args, imio_args):
+        """Get writers and frames.
+
+        Helper function for save_animation.
+        """
+        scene_gen = self._scene_gen
+        writers = {}
+        frames = {}
+
+        scene_gen = self._scene_gen
```

Comment:
this was already defined 4 lines above

Reply:
Oops, thanks, something went wrong in refactoring.  Fixed.